front50-test/src/main/groovy/com/netflix/spinnaker/front50/pipeline/PipelineDAOSpec.groovy

```diff
@@ -41,4 +41,37 @@ abstract class PipelineDAOSpec<T extends PipelineDAO> extends Specification {
     thrown IllegalArgumentException
   }
 
+  def "filters pipelines by pipeline name"() {
+    when:
+    instance.create("0", new Pipeline(application: "foo", name: "pipelineName1"))
+    for (int i = 1; i < 10; i++) {
+      def name = i % 2 == 0 ? "pipelineNameA" + i : "pipelineNameB" + i;
+      instance.create(i.toString(), new Pipeline(application: "foo", name: name))
+    }
+    def filteredPipelines = instance.getPipelinesByApplication("foo", "NameA", true);
+    filteredPipelines.sort()
```

Comment:
Don't think we need this line.  Tests pass for me without it.

Reply:
For some reason, I got error before but not anymore!! So removed it now.